docs/guide/laravel.md

```diff
@@ -204,6 +204,145 @@ Content-Type: application/json
 Controller-Method: ArticleController@getList
 Model-Method: Article::getList
 ```
+  
+## Performance Tuning SQL Queries
+
+SQL tuning is the process of improving SQL queries to accelerate your servers performance.
+It's general purpose is to reduce the amount of time it takes a user to 
+receive a result after issuing a query, and to reduce the amount of 
+resources used to process a query. 
+
+Big SQL queries can suffer performance issues. 
+Performance degradation often occurs if the database is not being properly 
+maintained or if queries can be rewritten more efficiently.
+
+Following Query optimization tips for better performance
+
+- **Apply Index On Necessary Columns:**
```

Comment:
@we-vikram-wri231 use `####` for all headings

Reply:
Ok @themodernpk Sir
I will update all changes